assets/css/detours/_detour_map.scss

```diff
@@ -15,3 +15,18 @@
   stroke: $color-strawberry-400;
   fill: $color-strawberry-400;
 }
+
+.c-detour_map--tooltip::before {
+  border-top-color: black;
+}
+
+.c-detour_map--tooltip {
+  background-color: black;
+  border-color: black;
+  color: white;
+  opacity: 100% !important;
+  font-size: 14px;
+  padding: 4px 8px 4px 8px;
+  font-family: var(--bs-body-font-family);
+  border-radius: var(--bs-border-radius);
+}
```

Comment:
**suggestion:** you could get at least some of this styling from just importing the Bootstrap Tooltip styling (uncommenting like 52 in `css/bootstrap.scss`). You could probably also get some more of the styling by updating Bootstrap SCSS variables rather than writing your own CSS rules (though some may still be necessary).